packages/modular-scripts/src/build/buildPackage/maybeCopyRootLicense.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import globby from 'globby';
+import getPrefixedLogger from '../../utils/getPrefixedLogger';
+import getModularRoot from '../../utils/getModularRoot';
+
+const licenseGlob = 'LICEN@(C|S)E*';
+
+export async function maybeCopyRootLicense(
+  target: string,
+  targetOutputDirectory: string,
+) {
+  const logger = getPrefixedLogger(target);
+  const modularRoot = getModularRoot();
+  const matches = globby.sync(path.join(targetOutputDirectory, licenseGlob), {
+    cwd: modularRoot,
+    onlyFiles: true,
+  });
+  if (matches.length == 0) {
```

Comment:
nit: `===`

Reply:
If the target package contains a license file, would this have been copied over by this point?

Reply:
Yeah, due to pack-list and the default files array containing LICENSE. It's worth adding that if a user is using a different license format they would have to add it to files manually.

Reply:
Unless you think modular should try and detect the license in that case too?